web/test/pages/profile-foreign.test.tsx

```diff
@@ -0,0 +1,489 @@
+/* eslint-disable jest/expect-expect */
```

Comment:
It might be nice to nest the profile tests in their own directory.

`web/test/pages/profile/*`